### PR TITLE
gui fixes

### LINKED ISSUE #1221: Inconsistent tool strip heights
All tool strips have a height of (25px) except for the main file menu (24px);
Therefor one grip dot is missing on the main tool strip
image
Also you can see the toolstrip container shifting one pixel when putting another toolstrip besides the main one:

I'll provide a PR for this ASAP

### PR DESCRIPTION
**Pull request by farosch**
See #1221

## Description
Fixed #1221 and provided some little extra "fixes" like the texts in the commits say.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: mRemoteV1/UI/Forms/OptionsPages/CredentialsPage.Designer.cs [hunk 1]
```diff
@@ -166,7 +165,6 @@ private void InitializeComponent()
             this.AutoScaleMode = System.Windows.Forms.AutoScaleMode.Font;
             this.Controls.Add(this.pnlDefaultCredentials);
             this.Name = "CredentialsPage";
-            this.PageIcon = ((System.Drawing.Icon)(resources.GetObject("$this.PageIcon")));
```
[the remark this answers is not in the document]
Comment (farosch) [reply mRemoteV1/UI/Forms/OptionsPages/CredentialsPage.Designer.cs]: I have pulled your latest changes from the develop branch and included them in this branch. The unhandled exception should be gone now.

[the remark this answers is not in the document]
Comment (sparerd) [reply mRemoteV1/UI/Forms/OptionsPages/CredentialsPage.Designer.cs]: The CredentialsPage still needs its PageIcon property set to the correct icon (mRemoteNG\mRemoteV1\Resources\Icons\FamFamFam\shield.ico). Re-adding the above highlighted line will fix the issue.

Without that, we get the default icon as shown here:
image

[the remark this answers is not in the document]
Comment (farosch) [reply mRemoteV1/UI/Forms/OptionsPages/CredentialsPage.Designer.cs]: The Credentials page does not have an Icon in its resource file, neither do AppearancePage, ConnetionsPage or ThemePage. They seem to set their icon with `PageIcon = Resources.Panels_Icon;` in their own public method. What is the proper way to do this here? Should I add the Icons for each option page to their resource file or should they all set theit icon in the public constructor?

[the remark this answers is not in the document]
Comment (sparerd) [reply mRemoteV1/UI/Forms/OptionsPages/CredentialsPage.Designer.cs]: Lets use the `PageIcon = Resources.Panels_Icon;` style as seen in the AppearancePage constructor. That seems much easier to follow than the designer generated value.

[the remark this answers is not in the document]
Comment (farosch) [reply mRemoteV1/UI/Forms/OptionsPages/CredentialsPage.Designer.cs]: alright, on it

[the remark this answers is not in the document]
Comment (sparerd) [reply mRemoteV1/UI/Forms/OptionsPages/CredentialsPage.Designer.cs]: Looks good now :+1:

**Comment by sparerd** [comment]
Fixes #990

**Pull request merged**